testdata/hybrid/memcached-operator/go.mod

```diff
@@ -3,31 +3,22 @@ module github.com/example/memcached-operator
 go 1.19
 
 require (
-	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/ginkgo/v2 v2.6.0
+	github.com/onsi/gomega v1.24.1
 	github.com/operator-framework/helm-operator-plugins v0.0.11
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v0.25.0
-	sigs.k8s.io/controller-runtime v0.13.0
+	k8s.io/apimachinery v0.26.0
```

Comment:
Bump to 0.26.2 or be consistent with library code at 0.26.1?

Reply:
We should be good to bump to v0.26.2, I'll do that

Reply:
I think this particular case is that the go.mod that is scaffolded is coming through via the go plugin in Kubebuilder. This would mean that in order to bump these to 0.26.2 we would need to make that change in Kubebuilder and then pull it in here.

IMO this should be fine as is since there is changes across multiple repos that would be needed.

Reply:
I misread @grokspawn's comment assuming that the comment is about `go.mod` for the helm operator and not for the test scaffolds, my bad. Yea, I bumped the test scaffolds to `v0.26.2` and noticed CI failures and as @everettraven mentioned we need to bump go plugin in KB first and then pull those here. I can work on the plugin k8s bump in follow-up PRs.